css/style.css

```diff
@@ -0,0 +1,180 @@
+/* * {
+  border: black 1px solid;
+} */
+* {
+  font-family: Helvetica;
+}
+
+#top-nav ul {
+  width: 98vw;
+  height: 20vh;
+  display: grid;
+  grid-template-columns: 20% 20% 20% 20% 20%;
+  grid-template-rows: 20% 20% 20% 20% 20%;
+  background-color: black;
+}
+
+li {
+  list-style: none;
+  color: grey;
+}
+
+#nav-Startrly {
+  grid-column-start: 1;
+}
+
+#nav-Try {
+  grid-column-start: 4;
+  text-align: right;
+}
+
+#nav-About {
+  grid-column-start: 5;
+  text-align: center;
+}
+
+#top-banner {
+  display: flex;
+  flex-direction: column;
+  background: url("https://nu.aeon.co/images/5f346448-1d3c-4e9a-a60b-7d485b58b2b0/header_Unseen-image-2-edit-2-1200x848.jpg");
+  color: #00e6ac;
+  margin: -20px;
+}
+
+#top-banner div:first-child{
+  font-size: 1.5em;
+  text-align: center;
+  vertical-align: middle;
+  margin-bottom: -30px;
+}
+
+#top-banner div:nth-child(2) {
+  font-size: 2.5em;
+  text-align: center;
+  vertical-align: middle;
+  margin-bottom: 30px;
+  margin-top: -15px;
+  font-weight: bold;
+}
+
+#top-banner a:link{
+    color: grey;
+    text-decoration: none;
+}
+
+#top-banner a:visited{
+    color: grey;
+    text-decoration: none;
+}
+
+#top-banner div:last-child{
+  font-size: 1.5em;
+  text-align: center;
+  vertical-align: middle;
+  margin-bottom: 100px;
+  margin-top: 5px;
+}
+
+
+#why-list, #footer-links {
```

Comment:
DRY CSS code: Check!  You used selectors to select multiple blocks of code at the same time and apply the same styles to them instead of repeating the same code for multiple blocks.